RobotsWantedLeague/Services/RobotsService.cs

```diff
@@ -1,59 +1,50 @@
+using Microsoft.Extensions.Options;
+using MongoDB.Driver;
+
 namespace RobotsWantedLeague.Services;
 
 using RobotsWantedLeague.Models;
 
 public class RobotsService: IRobotsService
 {
-    private readonly List<Robot> robots;
-    private int idGenerator = 0;
-    public List<Robot> Robots { get => robots; }
+    private readonly IMongoCollection<Robot> robotsCollection;
+
+    public List<Robot> Robots { get => robotsCollection.AsQueryable().ToList(); }
```

Comment:
To return the list of all the robot we need to first fetch them.  `robotsCollection.AsQueryable()` returns an object which is queryable via linq.  The query here is empty (aka take all) and the method to realize the query is `ToList`.